### PR TITLE
Archive WG App dev, kubeadm adoption, and container identity

### PR DESCRIPTION
**Pull request by castrojo**
Part of #3188

### CODE REVIEW

**Comment by castrojo** [comment]
/hold cancel

Yep, leads have already closed these down, just cleaning up the repo. Thanks!

File: sig-auth/README.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ The following subprojects are owned by sig-auth:
   - Owners:
     - https://raw.githubusercontent.com/kubernetes/kubernetes/master/staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/OWNERS
     - https://raw.githubusercontent.com/kubernetes/kubernetes/master/staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/OWNERS
-- **node-identity-and-isolation**
+- **node-n-and-isolation**
```
Comment (mikedanese) [inline sig-auth/README.md]: :cry:

Comment (cblecker) [reply sig-auth/README.md]: Was this a mistake @mikedanese?

Comment (mikedanese) [reply sig-auth/README.md]: Yes, looks like a sed was a bit over aggressive with replacing identity

Comment (cblecker) [reply sig-auth/README.md]: fix: https://github.com/kubernetes/community/pull/3730

**Pull request merged**